src/Elmish.WPF/BindingInternal.fs

```diff
@@ -14,7 +14,7 @@ type internal OneWayToSourceData<'model, 'msg, 'a> =
   { Set: 'a -> 'model -> 'msg }
 
 
-type internal OneWaySeqLazyData<'model, 'a, 'b, 'id when 'id : equality> =
+type internal OneWaySeqLazyData<'model, 'a, 'b, 'vm, 'id when 'id : equality> =
```

Comment:
This type parameter is also not being used, right?  So it can also be removed, right?

Reply:
I'll have to readd them in a later PR.

Reply:
Are any of the added type parameters being used in this commit?

Reply:
Yes the SubModel ones. The collection ones require a change to `CollectionTarget`

Reply:
The only one that *never* has a type is the `Cmd`

Reply:
Look up #473 if you want to see it in the final form. I'm not sure I'm going to be able to reshuffle stuff.

Reply:
Resolved in new version of commits.

Reply:
Unresolved as a reminder to me to check this